src/components/Todo.tsx

```diff
@@ -0,0 +1,40 @@
+import { FC } from 'react';
+import classNames from 'classnames';
+import { Todo as TodoType } from '../types/Todo';
```

Comment:
Using an alias, the same name of component and type is a little confusing to understand